codecov.yml

```diff
@@ -1,6 +1,12 @@
+# github_checks:
+#     annotations: false  # remove annoying inline code comments
```

Comment:
Did we want to keep the annotations? I have not seen them in action yet, could you point me to a pull request where they are shown?

Reply:
here's one https://github.com/ESMValGroup/ESMValCore/pull/1240/files - you can press "a" to toggle them on/off

Reply:
Thanks! @ESMValGroup/esmvaltool-coreteam Anyone an opinion on these type of comments by codecov? Are they helpful or too much? My feeling is that they are a bit too much in your face and that it may not be obvious to everyone that it is easy to hide them when reviewing.

Reply:
Just in case you missed the discussion on the topic above: @jvegasbsc and I expressed support for keeping the annotations. My main argument is that they shouldn't interfere with review, because at review stage there should be no complaints about uncovered lines in the diff left. As an author, they are really helpful to easily see where the tests need to be improved before asking for review. And if they disturb you too much, just press `a`.

Reply:
yeah pressing the key "a" does the trick, unless of course you have a French keyboard, in which case that'll mean Revolution :fr: